hpolib/abstract_benchmark.py

```diff
@@ -32,6 +32,11 @@ def __init__(self, rng: Union[int, np.random.RandomState, None] = None):
 
         self.rng = rng_helper.get_rng(rng=rng)
         self.configuration_space = self.get_configuration_space()
+        self.fidelity_space = self.get_fidelity_space()
+
+    @property
```

Comment:
Why do you use @property here?
What is the use case for this function? 

Perhaps you could rename it to get_default_fidelities()

Reply:
I think the default fidelity (or the fidelity to be optimized for in multi-fidelity opt) fits the role of a property more than a call to a one-line function. Of course, the difference between the two essentially boils down to saving the effort of typing two brackets when making a function call that takes no arguments and is also never expected to take any arguments.

Reply:
My original intention was to make it a class property, but that would involve some tinkering with the metaclass, so I opted to choose the simpler approach for now and shelve the idea for later.

Reply:
Defining it as a property in `AbstractBenchmark` itself has an additional advantage: it establishes a "default" relationship between the `default_configuration()` method of `get_configuration_space()` and the actual default fidelity used by any given benchmark, but which might be overwritten by any benchmarks, if they need to do so. I think I should rename it to `default_fidelity` though.

Reply:
We don't really use props in the rest of the code, so I would be in favour of making this consistent with `get_configuration_space()`, `get_fidelity_space()`

Reply:
Alright. Since this is a convenience method anyways, I would then just delete the property and default to using class.get_fidelity_space().get_default_configuration() whenever needed.